.github/workflows/build_and_test.yaml

```diff
@@ -56,6 +56,8 @@ jobs:
       run: cargo install cargo-hack --version 0.5.8
     - name: Check `cargo fmt` was run
       run: cargo fmt --all -- --check
+    - name: Check that the project is releaseable
+      run: shotover-proxy/build/is_releasable.sh
```

Comment:
you wont be able to run this script on every PR since the logic of the script depends on the release tag being on the current commit.

Reply:
Yep, just realised.